src/api/guardian.js

```diff
@@ -0,0 +1,17 @@
+const guardian = require("guardian-js");
+require("env2")("./config.env");
+const guardian_key = process.env.GUARDIAN_KEY;
+
+const guardianKey = (query, cb) => {
+  let api = new guardian(guardian_key, false);
```

Comment:
`new guardian` is a constructor so it needs capitalized `new Guardian` change it in at the top also when you require it

Reply:
also why is a let? is it gonna change later on ? im guessing no

also it's called `api` ( the constant ) that's a bad variable name

Reply:
It's the way recommended from the documentation https://github.com/PorterK/GuardianJSClient

Reply:
https://github.com/nervnews/nerv/pull/21#discussion_r161478572